src/components/Navbar/Navbar.js

```diff
@@ -0,0 +1,22 @@
+import React from "react"
+import "./Navbar.scss"
+import Logo from assets/skillChain.svg 
+
+
```

Comment:
Please merge in the current main branch into your ticket branch! Then, we are ready to merge!